task_interperator_ipf_server/__manifest__.py

```diff
@@ -33,7 +33,7 @@
     'depends': [
         'base_setup',
     ],
-    'installable': True,
+    'installable': False,
```

Comment:
If this module is something depricated, state it in the `name` field. 
Like `'name': 'DEPRICATED - IPF TLR Server'`